### PR TITLE
fix: improved markdown rendering

### LINKED ISSUE #747: Html in markdown is ignored
Any HTML code in markdown is currently ignored:

https://github.com/denoland/deno_website2/blob/dbbe3783bafdcdbd3d384efebeab8fb24b3ac373/components/Markdown.tsx#L124

Many repos rely on it for aligning, sizing etc, therefore entire sections of their readmes/docs are lost on `deno.land/x/`.

Is this something you plan to support eventually?

### PR DESCRIPTION
**Pull request by lucacasonato**
Markdown rendering is now heavily improved. All side-effect free HTML will now be rendered. The styling also matches GitHub's markdown rendering now.

Fixes #747

### CODE REVIEW

File: components/Markdown.tsx [hunk 1]
```diff
@@ -163,14 +58,106 @@ function Markdown(props: MarkdownProps) {
...
+    sanitizer: dompurify.sanitize,
+    highlight: (code, language) =>
+      renderToStaticMarkup(
+        <RawCodeBlock
+          code={code}
+          language={language as any}
+          disablePrefixes={true}
+          enableLineRef={false}
+        />
+      ),
```
Comment (lucacasonato) [inline components/Markdown.tsx:118]: This isn't great. I still need to ponder on how to do this without including multiple highlighters / themes. I will revisit this later.

**Comment by nayeemrmn** [comment]
Is it possible to get these changes on the home page as well? Inline code still looks not as clean for example. Seems like we have to used `marked()` there (for the main body of the page)

**Comment by lucacasonato** [comment]
> Inline code still looks not as clean for example

I'll fix that

File: components/Markdown.tsx (hunk 1 above)
Comment (ry) [reply components/Markdown.tsx:118]: ok

**Pull request merged**